pubspec.lock

```diff
@@ -1315,5 +1307,5 @@ packages:
     source: hosted
     version: "3.1.2"
 sdks:
-  dart: ">=3.1.0 <4.0.0"
-  flutter: ">=3.13.0"
```

Comment:
please revert this version change

Reply:
Changed on upgrading the dependencies in pubspec.yaml file.